library/core/src/fmt/mod.rs

```diff
@@ -570,8 +570,23 @@ impl Display for Arguments<'_> {
 /// There are a number of helper methods on the [`Formatter`] struct to help you with manual
 /// implementations, such as [`debug_struct`].
 ///
-/// `Debug` implementations using either `derive` or the debug builder API
-/// on [`Formatter`] support pretty-printing using the alternate flag: `{:#?}`.
+/// For simple cases, it's also possible to implement `Debug` using the [`write!`] macro:
+/// ```
+/// # use std::fmt;
+/// # struct Point {
+/// #     x: i32,
+/// #     y: i32,
+/// # }
+///
+/// impl fmt::Debug for Point {
+///     fn fmt(&self, f: &mut fmt::Formatter <'_>) -> fmt::Result {
+///         write!(f, "Point [{} {}]", self.x, self.y)
+///     }
+/// }
+/// ```
+///
+/// `Debug` implementations that have been created automatically by `derive` or implemented
+/// manually using a [`Formatter`] can be used by pretty-printing with the alternate flag: `{:#?}`.
```

Comment:
I feel like this sentence was more meaningful the way it was prior to the PR. *All* `Debug` implementations use a `Formatter` because that's dictated by the trait signature, and generally speaking *all* `Debug` implementations are either derived[^1] or manually implemented. So it's no longer clear what set of implementations is **not** covered by the description "created automatically by `derive` or implemented manually using a `Formatter`".

[^1]: macro-generated

Reply:
Reverted this.